portfolio/src/main/webapp/script.js

```diff
@@ -78,6 +78,7 @@ function createListElement(name, comment, postTime, sentimentScore) {
  * Fetches login status for the user and hides features for users not logged in.
  */
 function getLogin() {
+  document.getElementById('login-container').innerHTML = 'Loading...';
```

Comment:
if you're settings the text on this element, i think you can directly set textContent instead

Reply:
done.